.ci/travis-before-install.sh

```diff
@@ -1,6 +1,7 @@
 #!/bin/bash
 
-apt-get update -qq
-wget https://github.com/miloyip/rapidjson/archive/v1.1.0.tar.gz -O /tmp/v1.1.0.tar.gz
-tar -xvf /tmp/v1.1.0.tar.gz
-cp -r $PWD/rapidjson-1.1.0/include/rapidjson /usr/include/
+if [[ -z ${TOXENV} ]]; then
+    docker pull python:3.6
+    docker pull tendermint/tendermint:0.13
+    docker pull mongo:3.4.3
```

Comment:
Why pulling images here? I thought this was already covered by the `docker-compose` file.

Reply:
Oh yes, sure. I added this when I was troubleshooting to separate more clearly the different steps. Will take care of it.